packages/client-core/src/systems/ui/MediaControlsUI.tsx

```diff
@@ -87,7 +106,13 @@ const MediaControlsView = (props: MediaControlsProps) => {
             background-color: grey;
         }`}
         </style>
-        {mediaComponent.paused.value ? <PlayArrow style={{ fill: 'white' }} /> : <Pause style={{ fill: 'white' }} />}
+        const mediaStyles = { fill: 'white', width: `100%`, height: `100%` };
```

Comment:
we need to move this constant to the top after the const `mediaComponent` declaration line 61

Reply:
done